dev-bin/release.sh

```diff
@@ -63,8 +63,3 @@ git push
 gh release create --target "$(git branch --show-current)" -t "$version" -n "$notes" "$tag"
 
 git push --tags
-
-rm -fr build dist
-python -m sphinx -M html ./docs ./build/sphinx -W
```

Comment:
Does this still happen somehow?

Reply:
No, but I am not sure anyone is actually relying on it. It wouldn't show up in the binary wheels anyway. I would guess people are either using our online documentation or using the documentation shown within their editor.

If it turns out someone is using it, we can add it back, but I think I'd like to wait for some evidence of that first.